boot.py

```diff
@@ -13,3 +15,11 @@
 from .plugin.code_actions import *
 from .plugin.symbols import *
 from .plugin.rename import *
+
+
+def plugin_loaded():
+    main.startup()
```

Comment:
I find this to be too enterprisey, almost Java-esque. what can you possibly put here other than `main.startup`? Other than that, start to look good to me. Can't wait for this to get merged.

Reply:
I'm open to other suggestions for both `boot.py` and `startup/shutdown`! I renamed to `boot.py` as calling it `__init__.py` caused multiple instances of the packages to be started.